app/src/main/java/com/pajato/android/gamechat/database/GroupManager.java

```diff
@@ -187,6 +187,15 @@ public String getGroupProfilePath(final String groupKey) {
         return result;
     }
 
+    /** Return the group's common room */
+    public Room getCommonRoom(final String groupKey) {
+        // Return the group if it has been loaded.  Set a watcher to load it if not.
+        Group result = groupMap.get(groupKey);
```

Comment:
This code should be simplified along the lines: `return group != null ? RoomManager... : null;`